### PR TITLE
feature: defining of priorityClassName and add ServiceMonitor in helm chart

### LINKED ISSUE #99: Support of custom priorityClassName
It will be nice if you add define of custom priorityClassName in DaemonSet of kfo in helm chart.
For example in daemonset.yaml:
```
{{- if .Values.priorityClassName }}
      priorityClassName: "{{ .Values.controller.priorityClassName }}"
```
values.yaml:
```
priorityClassName: "customCriticalClass"
```

### LINKED ISSUE #100: Support generating of ServiceMonitor
Please add feature of generating ServiceMonitor in helm chart.
For example, in values.yaml
```
prometheusEnabled: true
ServiceMonitorEnabled: true
```

### PR DESCRIPTION
**Pull request by evilrussian**
This PR makes avaiable defining priorityClassName in daemonSet of kube-fluentd-operator in helm chart.
And it adds ServiceMonitor for supporting of Prometheus Operator 

closes #99 
closes #100

### CODE REVIEW

**Comment by evilrussian** [comment]
@jvassev ?

**Comment by gautam2187** [comment]
@evilrussian Can you please add the DCO as suggested by the bot.

Thanks!

**Comment by evilrussian** [comment]
@gautam2187 
I don't understand what i need to do. This PR has 2 commits and all commits are Signed-off

File: log-router/templates/servicemonitor.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{{- if .Values.serviceMonitor.enabled }}
```
Comment (gautam2187) [inline log-router/templates/servicemonitor.yaml:1]: @evilrussian should we also check for prometheusEnabled here with serviceMonitor?

**Comment by gautam2187** [comment]
@evilrussian I think the bot is expecting "Signed-off-by: John Doe <john.doe@email.org>" in the commit message.

File: log-router/templates/servicemonitor.yaml (hunk 1 above)
Comment (evilrussian) [reply log-router/templates/servicemonitor.yaml:1]: Yes. If prometheusOperator is not installed, then CRD ServiceMonitor will not be defined in cluster, and you will get an error 
`error: unable to recognize "ServiceMonitor.yaml": no matches for kind "ServiceMonitor" in version "monitoring.coreos.com/v1`

**Comment by evilrussian** [comment]
@gautam2187 all check have passed

**Pull request merged**